src/udp_wrap.cc

```diff
@@ -219,6 +221,35 @@ void UDPWrap::Bind6(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void UDPWrap::BufferSize(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  UDPWrap* wrap;
+  ASSIGN_OR_RETURN_UNWRAP(&wrap,
+                          args.Holder(),
+                          args.GetReturnValue().Set(UV_EBADF));
+
+  CHECK(args[0]->IsUint32());
+  CHECK(args[1]->IsUint32());
+  int size = static_cast<int>(args[0].As<Uint32>()->Value());
```

Comment:
It's not trivial since JS numbers are system independent, but C `int` _is_ system dependent.
IMHO simplest solution is throwing `if (size != args[0].As<Uint32>()->Value()) return env->ThrowUVException(FIND_A_GOOD_ERRNO, "uv_recv_buffer_size");`